README.md

```diff
@@ -109,3 +109,56 @@ Formatting will be run via the default nix command. You can find the formatter c
 ```bash
 nix fmt
 ```
+
+#### Contribution Guide
```

Comment:
I added a guide to adding a new package, that documents all the steps, cause it can be kinda confusing with the new directory structure.